db/migrate/20220429051355_add_user_ref_to_likes.rb

```diff
@@ -1,5 +1,6 @@
 class AddUserRefToLikes < ActiveRecord::Migration[7.0]
   def change
-    add_reference :likes, :user, null: false, foreign_key: true
+    add_reference :likes, :user, null: false, index: true
+    add_foreign_key :likes, :users, column: :author_id, on_delete: :cascade
```

Comment:
Same here 

![image](https://user-images.githubusercontent.com/65068771/167688136-f555b282-3d74-4d74-b07e-4468ff1329c8.png)


- [ ] Kindly correct this migration as is throwing an error. :point_up: 
Check this example :point_down: 

```rb 
class AddUserRefToLikes < ActiveRecord::Migration[7.0]
  def change
    add_reference :likes, :author, null: false, index: true, foreign_key: { to_table: 'users' }
  end
end
```